public/index.html

```diff
@@ -1,16 +1,17 @@
 <!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <meta name="theme-color" content="#000000">
-    <!--
+<html lang="en" style="max-width:100%; overflow-x:hidden;">
```

Comment:
Why are adding these attributes to the `html` and `body` tags?

Reply:
It's a fix I found online to prevent horizontal scrolling!